app/main/controllers/review-submission-ctrl.js

```diff
@@ -1,8 +1,177 @@
 'use strict';
-angular.module('myApp')
-.controller('ReviewSubmissionCtrl', function ($scope, $state) {
+ angular.module('myApp')
+.controller('ReviewSubmissionCtrl', function ($scope, $http, $state, DataService, $document,$uibModal, $log, $rootScope, $filter, SignaturePad) {
 
+var $ctrl = this;
 
+  $scope.submit = function(isValid) {
 
+      $scope.submitted = true;
 
+      if( $rootScope.application.request_content.review.signature !== undefined) {
+
+            console.log("cash money");
+        //  $state.go('home');
+      }
+        else if (isValid == false) {
+          console.log("Valid Check failed");
+
+      }
+          else {
+        $ctrl.open();
+      }
+
+    };
+
+
+  $ctrl.animationsEnabled = true;
+
+  $ctrl.open = function (size, parentSelector) {
+    var parentElem = parentSelector ?
+      angular.element($document[0].querySelector('.modal-demo ' + parentSelector)) : undefined;
+    var modalInstance = $uibModal.open({
+      animation: $ctrl.animationsEnabled,
+      ariaLabelledBy: 'modal-title',
+      ariaDescribedBy: 'modal-body',
+      templateUrl: 'myModalContent.html',
+      controller: 'ModalInstanceCtrl',
+      controllerAs: '$ctrl',
+      size: size,
+      appendTo: parentElem,
+      resolve: {
+        items: function () {
+          return $ctrl.items;
+        }
+      }
+    });
+
+  };
+  $ctrl.items = ['item1', 'item2', 'item3'];
+
+
+  $ctrl.toggleAnimation = function () {
```

Comment:
Is this ever getting called? If not, animationsEnabled isn't needed either -- just hardcode the value to the .open call.